grid-proxy/pkg/client/grid_client.go

```diff
@@ -365,3 +435,25 @@ func (g *Clientimpl) newHTTPClient() *http.Client {
 		},
 	}
 }
+
+func (g *Clientimpl) prepareURL(path string, params ...interface{}) (string, error) {
+	values := url.Values{}
```

Comment:
var values url.Values? please apply for others

Reply:
it needs to be initialized